core/vm/evm.go

```diff
@@ -51,7 +51,7 @@ func (evm *EVM) precompile(addr common.Address) (PrecompiledContract, bool) {
 	case evm.chainRules.IsByzantium:
 		precompiles = PrecompiledContractsByzantium
 	default:
-		precompiles = PrecompiledContractsHomestead
+		precompiles = PrecompiledContractsBerlin
```

Comment:
why is Berlin the default here again?

Reply:
Again, I have to choose one of them as the default, it could be any of them.